src/screens/GroupInfo/index.js

```diff
@@ -281,15 +281,15 @@ const GroupInfo = () => {
             isOpen={openModal}
             onCloseModal={handleCloseSelectUser}
             selectedUser={selectedUser}
-            onPress={alertRemoveUser}
+            onPress={handleOpenPopup}
             isGroup={channelState?.channel?.data?.type === 'group'}
           />
           <ModalActionAnonymous
             name={selectedUser?.user?.anonymousUsername}
             isOpen={isAnonymousModalOpen}
             onCloseModal={handleCloseSelectUser}
             selectedUser={selectedUser}
-            onPress={alertRemoveUser}
+            onPress={handleOpenPopup}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onPress` prop for both `ModalAction` and `ModalActionAnonymous` components has been updated from `alertRemoveUser` to `handleOpenPopup`. This change aligns with the renaming of the function mentioned above. Ensure that the `handleOpenPopup` function has the same functionality as the previous `alertRemoveUser` or any differences are handled appropriately.
```diff
-            onPress={alertRemoveUser}
+            onPress={handleOpenPopup}
```
```diff
-            onPress={alertRemoveUser}
+            onPress={handleOpenPopup}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->